src/scroll-timeline-base.js

```diff
@@ -506,6 +506,7 @@ export function calculateRange(phase, container, target, axis, optionsInset) {
 
   let startOffset = undefined;
   let endOffset = undefined;
+  const targetIsTallerThanContainer = viewSize > containerSize ? true : false;
 
```

Comment:
Suggest we add a TODO here for handling position sticky. Naturally, we can implement position sticky in a follow up PR.

Reply:
Added.